common/src/main/java/com/sequenceiq/cloudbreak/common/mappable/CloudPlatform.java

```diff
@@ -12,11 +14,22 @@ public enum CloudPlatform {
     // DEPRECATED Platforms
     OPENSTACK;
 
+    private static final Map<CloudPlatform, String> DISPLAY_NAME = new HashMap<>() {
+        {
+            put(AWS, "Amazon Web Services");
+            put(AZURE, "Microsoft Azure");
```

Comment:
GCP is missing (yarn too...)

Reply:
Only for these, display name is different than name. For others, it will use name(). Please refer to getDislayName().

Reply:
GCP is Google Cloud Platform

Reply:
In all error messages, we use GCP only. I have added this when we have to provide cloud name as part of error message so that it's consistent across error messages.